nixos/maintainers/scripts/gce/create-gce.sh

```diff
@@ -1,15 +1,27 @@
-#! /bin/sh -e
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p google-cloud-sdk
 
-BUCKET_NAME=${BUCKET_NAME:-nixos-images}
-export NIX_PATH=nixpkgs=../../../..
-export NIXOS_CONFIG=$(dirname $(readlink -f $0))/../../../modules/virtualisation/google-compute-image.nix
-export TIMESTAMP=$(date +%Y%m%d%H%M)
+set -euo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+NIXPKGS_ROOT="$DIR/../../../../"
+BUCKET_NAME="${BUCKET_NAME:-nixos-images}"
+export NIX_PATH=nixpkgs="$NIXPKGS_ROOT"
+export NIXOS_CONFIG="$NIXPKGS_ROOT/nixos/modules/virtualisation/google-compute-image.nix"
+TIMESTAMP="$(date +%Y%m%d%H%M)"
+export TIMESTAMP
 
 nix-build '<nixpkgs/nixos>' \
-   -A config.system.build.googleComputeImage --argstr system x86_64-linux -o gce --option extra-binary-caches http://hydra.nixos.org -j 10
+   -A config.system.build.googleComputeImage \
+   --argstr system x86_64-linux \
+   -o gce \
+   --option extra-binary-caches https://hydra.nixos.org \
```

Comment:
I swapped http with https here, to make building an image remote a bit more trustworthy.